src/dfx/src/commands/ledger/show_subnet_types.rs

```diff
@@ -0,0 +1,38 @@
+use crate::lib::environment::Environment;
+use crate::lib::error::DfxResult;
+use crate::lib::ledger_types::{GetSubnetTypesToSubnetsResult, MAINNET_CYCLE_MINTER_CANISTER_ID};
+use crate::lib::waiter::waiter_with_timeout;
+use crate::util::expiry_duration;
+
+use crate::lib::root_key::fetch_root_key_if_needed;
+
+use anyhow::{anyhow, Context};
+use candid::{Decode, Encode};
+
+const GET_SUBNET_TYPES_TO_SUBNETS_METHOD: &str = "get_subnet_types_to_subnets";
+
+pub async fn exec(env: &dyn Environment) -> DfxResult {
+    let agent = env
+        .get_agent()
+        .ok_or_else(|| anyhow!("Cannot get HTTP client from environment."))?;
+
+    fetch_root_key_if_needed(env).await?;
+
+    let result = agent
+        .update(
+            &MAINNET_CYCLE_MINTER_CANISTER_ID,
```

Comment:
Can you add a `--cmc-canister-id` flag in case someone has the CMC installed in a different place? You can copy/paste from `dfx ledger transfer --ledger-canister-id`

Reply:
Done.